packages/angular/src/table/head/table-head.scss

```diff
@@ -0,0 +1,31 @@
+@import '~carbon-components/scss/globals/scss/vars';
```

Comment:
This has a high chance of blowing up in different envs, and the PAL already proxies `carbon-components` so we shouldn't need both this and the PAL import